test/pkg/distp/distp.go

```diff
@@ -19,4 +19,5 @@ const (
 	DistAPIVersionValue string = "registry/2.0"
 
 	UploadUuidKey string = "Docker-Upload-Uuid"
```

Comment:
These changed recently. Not sure to test against the docker values as well. Perhaps this could be a struct, to then have an array of them to iterate over.